RFC/src/RFC-0310_AssetTemplates.md

```diff
@@ -0,0 +1,218 @@
+| initial_state_hash                 | `u256`        | The hash of the canonical serialisation of the initial template state (of the template-specific data)                         |
+| **Signatures**                     |               |                                                                                                                               |
+| creator_sig                        | `u256`        | A digital signature of the preceding 2 sections’ data using the asset creator’s private key corresponding to the `issuer` PKH |
+| **template-specific data**         |               | Depending on the template, the data in the final section varies and corresponds to the initial asset state                    |
+
+#### Committee parameters
+
+If `committee_mode` is `PERMISSIONED` the `committee_mode` object is
+
+| Name             | Type         | Description |
+|:-----------------|:-------------|:------------|
+| trusted_node_set | Array of PKH | See below   |
+
+Only the nodes in the trusted node set will be allowed to execute instructions for this asset.
+
+If `committee_mode` is `PERMISSIONLESS` the `committee_mode` object is
+
+| Name                    | Type  | Description                                                                                                                       |
+|:------------------------|:------|:----------------------------------------------------------------------------------------------------------------------------------|
+| node_threshold          | `u32` | The total number of validator nodes that can register to execute instructions for this asset                                      |
+| minimum_collateral      | `u64` | The minimum amount of Tari a validator node must put up in collateral in order to execute instructions for this asset.([RFC-???]) |
+| node_selection_strategy | `u32` | The selection strategy to employ allowing nodes to register to manage this asset                                                  |
+
+
+#### Issuer
+
+Anyone can issue assets on the Tari network from their wallet or client program. The wallet will provide the PKH and
+sign the instruction. The wallet needn’t use the same private key each time.
+
+#### Name and Description
+
+These fields are purely for informational purposes and do not need to be unique, and do not act as an asset ID.
+
+#### Top level digital issuers
+
+If it is likely that a digital asset issuer will be issuing many assets on the Tari Network (hundreds, or thousands),
+the issuer should strongly consider registering a name (e.g. `TARILABS`) as a top-level digital asset issuer. This is
+done via a special transaction on the base layer, described in [RFC-???](). A registered TLDI prevents spoofing of
+assets from copy cats or other malicious actors. It also makes asset discovery simpler.
```

Comment:
Change "copy cats" to "copycats" -> https://dictionary.cambridge.org/dictionary/english/copycat